src/async.js

```diff
@@ -27,6 +28,17 @@ Async.prototype.setScheduler = function(fn) {
     return prev;
 };
 
+Async.prototype.setTimeoutScheduler = function(setTimeoutFn) {
+    var prev = this._setTimeout;
+    this._setTimeout = setTimeoutFn;
+    return prev;
```

Comment:
because a) this makes it consistent with setScheuler API and b) it makes it easy to unit test, with this unit tests can  set custom value and restore it when testing done. Actually I do use this feature in the uni test.